lib/compress.js

```diff
@@ -4827,6 +4828,11 @@ merge(Compressor.prototype, {
                 && is_object(self.left.fixed_value())) {
                 return make_node(self.operator[0] == "=" ? AST_True : AST_False, self);
             }
+            // void 0 == x => null == x
+            else if (!is_strict_comparison
+                && is_undefined(self.left, compressor)) {
```

Comment:
Style preference - I think these two conditions can be on one line since it's short.

Reply:
Please move this code immediately below the `case "==": case "!=":` lines.

Reply:
> Please move this code immediately below the case "==": case "!=": lines.

Sure. Why?

Reply:
So it's closer to the `is_strict_comparison` variable definition. Makes code easier to follow.

Reply:
Makes sense.